evalml/tests/model_understanding_tests/prediction_explanations_tests/test_algorithms.py

```diff
@@ -49,8 +48,6 @@ def __init__(self, parameters, random_seed=0):
 
 
 baseline_message = "You passed in a baseline pipeline. These are simple enough that SHAP values are not needed."
-xg_boost_message = "SHAP values cannot currently be computed for xgboost models."
```

Comment:
I didn't see this message being used anywhere. I think its left over from a previous change.